easyrest/models/menu.py

```diff
@@ -0,0 +1,26 @@
+"""
+This module describe data model for "menu" table
+"""
+
+from sqlalchemy import (
+    Column,
+    Integer,
+    Text,
+    ForeignKey,
+)
+from sqlalchemy.orm import relationship
+
+from .meta import Base
+
+
+class Menu(Base):
+    """
+    The data model of "menu" table
+    Defines data structure of "menu" table
+    Relationship: menu -> menu_item
+    """
+    __tablename__ = 'menu'
```

Comment:
All table names should be in plural form: e.g. "menus". Model class name is correct here and should be in singular. Please update all models in this PR.